AndroidSDKCore/src/main/java/com/leanplum/internal/RequestSequence.java

```diff
@@ -0,0 +1,18 @@
+package com.leanplum.internal;
+
+/**
+ * Public interface for during the call sequence of database
```

Comment:
javadoc: `public interface` is redundant.

Now that we have written the test I think a better name for this interface can be `RequestSequenceRecorder`.

javadoc: Records request call sequence of read/write operations.

Reply:
done